ui/StatusQ/src/StatusQ/Popups/StatusSearchLocationMenu.qml

```diff
@@ -72,7 +72,7 @@ StatusMenu {
                     assetSettings.name: !!model.imageSource ? !!model.imageSource : model.iconName
                     assetSettings.isImage: !!model.imageSource
                     assetSettings.isLetterIdenticon: !model.imageSource && !model.iconName
-                    assetSettings.imgIsIdenticon: model.isIdenticon
```

Comment:
`isIdenticon` no longer exists?

Reply:
as I understand, its no longer exists in search model, I checked the model and can't find `isIdenticon` role (its can be some copy/paste)